bundle/config/resources/apps.go

```diff
@@ -23,6 +23,10 @@ type AppPermission struct {
 }
 
 type App struct {
+	// This is app's name pulled from the state. Usually the same as Name but may be different if Name in the config
+	// was changed but the app was not re-deployed yet.
+	ID string `json:"id,omitempty" bundle:"readonly"`
```

Comment:
Apps now have their own Id field as part of App structure, how do these two play together?

Reply:
Good catch. apps.App has Id field, so they are distinct but they both serialize to "id", so one would shadow the other. 

This is a general problem -- any resource can have ID/id field and we risk overlap in bundle summary.

Reply:
Discussed offline - we'll use CreateAppRequest for apps (in a separate PR). I think this one should be safe to merge though, Apps.Id is always empty.